github/github.go

```diff
@@ -154,6 +156,19 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// MediaRawType represents raw return of a request instead of json format
+type MediaRawType int
+
+const (
+	Diff MediaRawType = 1 + iota
+	Patch
+)
+
+// GetRawOptions specifies the optional parameter when user want to get raw presentation of a response instead of json format
```

Comment:
I think you can simplify the docs to just:

```
// GetRawOptions specifies the parameters to the PullRequestsService.GetRaw method.
```

The parameter is not optional, so it's not accurate to say that it is.